app/checks-tests/free_ram_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo " >> Assert that 2GB free (3/5GB used) is ok when max usage is 70%"
+MOCK_FREE_RAM=3000 MOCK_TOTAL_RAM=5000 RAMPERCENT=70 ../checks/free_ram.py
+[[ $? == 1 ]] && echo " .. Test failed." && exit 1
+[[ $? == 127 ]] && echo " .. Test lacking a parameter." && exit 127
```

Comment:
I don't know if this would work. `$?` is generally a last command exit result, it may be eg. taking `[[ $? == 1 ]] && echo " .. Test failed." && exit 1` as a single command for example.

Reply:
fixed